devices/sunricher.js

```diff
@@ -3,8 +3,41 @@ const fz = {...require('../converters/fromZigbee'), legacy: require('../lib/lega
 const tz = require('../converters/toZigbee');
 const reporting = require('../lib/reporting');
 const extend = require('../lib/extend');
+const utils = require('../lib/utils');
 const e = exposes.presets;
 
+const fzLocal = {
+    sunricher_SRZGP2801K45C: {
+        cluster: 'greenPower',
+        type: ['commandNotification', 'commandCommisioningNotification'],
+        convert: (model, msg, publish, options, meta) => {
+            const commandID = msg.data.commandID;
+            if (utils.hasAlreadyProcessedMessage(msg, msg.data.frameCounter, `${msg.device.ieeeAddr}_${commandID}`)) return;
+            if (commandID === 224) return;
+            const lookup = {
+                0x21: 'press_on',
+                0x20: 'press_off',
+                0x37: 'press_high',
+                0x38: 'press_low',
+                0x35: 'hold_high',
+                0x36: 'hold_low',
+                0x34: 'high/low_release',
+                0x63: 'CW/WW_release',
+                0x62: 'CW_dec/WW_inc',
+                0x64: 'WW_inc/CW_dec',
+                0x41: 'R>G>B',
+                0x42: 'B<G<R',
+                0x40: 'RGB_release',
```

Comment:
It is preferred to keep all actions snake case (lower case) (e.g. `RGB_release` -> `rgb_release`)